Modern_Tides.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
@ryan-rutishauser Unfortunately this was placed in the wrong location - please move to `src/_data/styles/` in order for your submission to be counted, thanks!